### DISCUSSION TITLE
Wlan0 does not go into monitor mode since release 2.x

### DISCUSSION THREAD

**Discussion by dafunkychild** [WLAN-Pi/feedback#6] [General Inquiries] [answered]
Hello,

I use Wlanpi with WiFi USB dongle COMFAST 912AC.

Up to release 1.9.1 I was able to turn wlan0 into monitor to capture frames but since release 2.0.0 it is no longer possible.

Here is the result with release 1.9.1

![Wlanpi-1 9Capture-20210323123658-1066x684](https://user-images.githubusercontent.com/57414097/112176592-8554a780-8bf8-11eb-83ef-6c588d6a5e14.png)

With relase 2.0.1

![Wlanpi-2 0 1](https://user-images.githubusercontent.com/57414097/112176813-b59c4600-8bf8-11eb-85c7-cee3674bfc2c.png)

After few kernel messages the wlanpi freezes.

**Comment by joshschmelzle** [comment, marked as answer]
Try using the `ip`/`iw` tools to stage the NIC in monitor mode prior to running various tools to perform captures (airdump, horst, etc).

Here is one example using `wlan0` as the desired interface:

```
sudo ip link set wlan0 down
sudo iw dev wlan0 set type monitor
sudo ip link set wlan0 up
```

**Reply by dafunkychild** [reply]
I should have posted in the issues section.

Thank you joshschmelzle, the cli you gave work like a charm.
